.circleci/config.yml

```diff
@@ -1234,17 +1340,18 @@ workflows:
                       - secrethub/env-export:
                             secret-path: graviteeio/cicd/graviteebot/git/user/email
                             var-name: GIT_USER_EMAIL
-            - nexus-staging-prepare-component:
-                  requires:
-                      - release
-                  context: cicd-orchestrator
-                  pre-steps:
-                      - secrethub/env-export:
-                            secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_access_key
-                            var-name: AWS_ACCESS_KEY_ID
-                      - secrethub/env-export:
-                            secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_secret_key
-                            var-name: AWS_SECRET_ACCESS_KEY
+            # - nexus-staging-prepare-component:
+            #       requires:
+            #           - Commit and prepare next version
+            #       context: cicd-orchestrator
+            #       pre-steps:
+            #           - secrethub/env-export:
+            #                 secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_access_key
+            #                 var-name: AWS_ACCESS_KEY_ID
+            #           - secrethub/env-export:
+            #                 secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_secret_key
+            #                 var-name: AWS_SECRET_ACCESS_KEY
+
```

Comment:
To remove?

Reply:
done in next commit. Disabled here for test